data/data/ibmcloud/master/outputs.tf

```
@@ -0,0 +1,3 @@
+output "ip_addresses" {
+  value = ibm_is_instance.master_node.*.primary_network_interface.0.primary_ipv4_address
```

I recommend that you use the full splat operator rather than the legacy one.
```suggestion
  value = ibm_is_instance.master_node[*].primary_network_interface[0].primary_ipv4_address
```

---

okay, will update

---

fixed